learn/cookbooks/vercel.mdx

```diff
@@ -0,0 +1,94 @@
+# Integrate Meilisearch Cloud with Vercel
+
+In this guide you will learn how to link a Meilisearch Cloud instance to your Vercel project.
+
+## Introducing our tools
+
+### What is Vercel?
+
+[Vercel](https://vercel.com/) is a cloud platform for building and deploying web applications. It works out of the box with most popular web development tools.
+
+### What is Meilisearch Cloud?
+
+[Meilisearch Cloud](https://www.meilisearch.com/cloud) is a cloud-based SaaS platform, built on top of the Meilisearch open-source search engine. It offers a managed search service that is scalable, reliable, and designed to meet the needs of all companies
```

Comment:
i think you can remove the "is a cloud-based saas platform, built on top of the meilisearch oss engine"

ie. "[Meilisearch Cloud](https://www.meilisearch.com/cloud) offers a managed search service that is scalable, reliable, and designed to meet the needs of all companies"

(missing period at the end of your sentence)

Reply:
also update add UTM params!